### ISSUE TITLE
IIIF card: Need to support multiple tiles, similar to digital dataset or photogallery card

### ISSUE THREAD

**Issue by dwuthrich** [archesproject/arches#5971]
The IIIF card will need to support creation of tile and navigation between tiles in a manner similar to the digital dataset (charting card) and photo gallery cards.  Users will expect a UX that allows them to rapidly move between tiles (e.g.: both display of a IIIF manifest canvas and the form associated with a specific annotation).

**Comment by robgaston** [comment]
@dwuthrich is the tree on the left not sufficient for this (changing between tiles)?  It will be a large departure I think to rework the card in this way, so we should probably talk this through with Cyrus to discuss the desired UX and if this rework would really be the best/most efficient way to support it

**Comment by dwuthrich** [comment]
No, the tree isn't sufficient for navigating between tiles, as the following gif shows.  There are at least two problems:

1. Creating a new tile from the tree re-initializes the card.  This means that a user would have to navigate back to the desired canvas and then zoom/pan to the spot that they want to annotate.

2. The UX to create new tiles or navigate between tiles isn't very good.

As shown in this gif, if a user wanted to add a new annotation near the one they've just completed, the steps required to do so are pretty onerous:  https://recordit.co/qr8HFqov9t

**Comment by robgaston** [comment]
Item 1 can probably be handled without making the change you suggested; in fact it would probably need to be handled as part of the card regardless of whether we make that change or not (and would need to be handled equally in either event).

Item 2 is perhaps a more general issue that we might consider how to improve in the resource editor more broadly.

I think that the suggested solution would not necessarily (or on it's own) address the UX concerns here.  It seems like much of the issue comes down to the need for the IIIF card to maintain its own state better (manifest/canvas selection, pan/zoom, maybe others?) across tile selection.  Adding a way to select tiles internal to the card would not on it's own address the issues w/ card state management.

**Comment by dwuthrich** [comment]
@robgaston Agree that this might be appropriately handled by improving the resource editor.  We should discuss possible solutions
